.golangci.yml

```diff
@@ -268,6 +268,10 @@ issues:
     - staticcheck
     text: "SA1019: in.(.+) is deprecated"
     path: .*(api|types)\/.*\/conversion.*\.go$
+  - linters:
+      - revive
+    text: "if-return: redundant if ...; err != nil check, just return error instead"
+    path: .*(api|types|test)\/.*\/conversion.*\.go$
```

Comment:
nit: can we add a comment explaining why we need these lines?

Reply:
Do you mean for each specific case? It's in the "Disable linters for conversion" section starting on [L266](https://github.com/kubernetes-sigs/cluster-api/blob/8799883765931ba519546da60112924725ca498f/.golangci.yml#L266-L290)